packages/synapse-react-client/src/components/CardContainerLogic/CardContainerLogic.stories.ts

```diff
@@ -113,11 +117,79 @@ export const PublicationCard: Story = {
   },
 }
 
+const genieSelectedFacetConfigs: SelectedFacetConfig[] = [
+  {
+    facetColumnName: 'cohort',
+    facetValueColumnName: 'Cohort',
+  },
+  {
+    facetColumnName: 'version',
+    facetValueColumnName: 'version',
+  },
+]
+
+export const ReleaseCardLarge: Story = {
```

Comment:
Demonstration of the GENIE release card buttons linking to filtered explore pages using an example source table. GENIE releases link to GENIE explore page. GENIE-BPC releases link to GENIE-BPC explore page. Both primary and secondary buttons filter to the selected cohort and version. The secondary button also filters to only data_guide files.

https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/7ff0b5d9-7617-4e78-80df-95c314905c01